apps/condo/lang/en.json

```diff
@@ -565,6 +566,21 @@
   "Seconds": "Seconds",
   "Update": "Update",
   "Updated": "Updated",
+  "LoginBySBBOL": "Login with SberBusiness ID",
+  "subscription.data.tariff": "Subscription plan",
+  "subscription.data.tariffs": "Subscription plans",
+  "subscription.data.cancelSubscription": "Cancel subscription",
+  "subscription.data.subscription": "Subscription",
+  "subscription.data.status": "Subscription status",
+  "subscription.data.accountsCount": "Number of personal accounts",
+  "subscription.data.value": "Total value",
+  "subscription.data.createBill": "Bill",
+  "subscription.alert.subscription.payment.pending": "An invoice request has been sent to the sales department, we will contact you shortly",
+  "subscription.option.description.clients": "For clients",
+  "subscription.option.description.account": "at personal expense",
+  "subscription.modal.complete.action": "OK",
+  "subscription.modal.complete.description": "We will contact you as soon as possible to clarify the number of premises and calculate the cost of the subscription.\nThe preliminary number of rooms is 25.",
+  "subscription.modal.complete.title": "Invoicing request accepted",
```

Comment:
We need a tool to sync translates line numbers

Reply:
I have an idea, how to implement it.
DOMA-1272

Reply:
We discuss this with @dkoviazin and @pahaz months ago;)